.github/workflows/tics.yaml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   CI:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
If I'm not mistaken, didn't @sergiusens pin this to 22.04 on purpose when building this workflow? python 3.10 perhaps?